src/components/FindMovie/FindMovie.tsx

```diff
@@ -17,41 +80,56 @@ export const FindMovie: React.FC = () => {
               id="movie-title"
               placeholder="Enter a title to search"
               className="input is-dander"
+              value={title}
+              onChange={onInputChangeHandler}
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {(isLoaded && movie === null) && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
         <div className="field is-grouped">
           <div className="control">
             <button
               data-cy="searchButton"
               type="submit"
-              className="button is-light"
+              className={classNames('button is-light', {
+                'is-loading': isLoading,
+              })}
+              onClick={onSearchHandler}
+              disabled={title === ''}
             >
-              Find a movie
+              {!isSearched
+                ? 'Find a movie'
+                : 'Search again'}
```

Comment:
In my opinion something like this would look better
```suggestion
              {
                !isSearched ? 'Find a movie' : 'Search again'
              }
```